CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+*              @fiendish @znatty22 @allisonheath 
+/scripts/      @fiendish @znatty22 @allisonheath @alubneuski
```

Comment:
Is including `@fiendish @znatty22 @allisonheath` in lines after `*` repetitive? Since `*` says require review for everything in the repo?

Reply:
Also, I made a new team. You can replace all occurrences of`@fiendish @znatty22 @allisonheath` with `@kids-first/data-ingest`

Reply:
> Is including @fiendish @znatty22 @allisonheath in lines after * repetitive?

I don't think so, because the template for the owners file says that subsequent overlapping rules override prior settings.

Reply:
Though I guess if I switch the line order then maybe.

Reply:
k 👍